databricks/sdk/service/iam.py

```diff
@@ -1182,7 +1182,8 @@ def list(self,
 
         # deduplicate items that may have been added during iteration
         seen = set()
-        query['startIndex'] = 0
+        query['startIndex'] = 1
```

Comment:
Is this because the first item starts from number 1 in query? https://docs.databricks.com/api/workspace/groups/list

Reply:
Yes, that's right.